dev-docs/RFCs/v7.x/spring-based-transitions-rfc.md

```diff
@@ -7,11 +7,40 @@
 
 ## Abstract
 
-[FILL ME IN]
+As previous animation- and transition-related RFCs have pointed out, easy-to-use animation and
+transition systems are often-requested features. The existing interpolation-based transition API
+allows users to define transitions much like you'd define one with CSS. Given a duration and 
+easing method, the transition system will interpolate between a `from` value and a `to` value.
+For natural-looking transitions, an easing method (e.g. `ease-in-out`) is often used to make the
+animating object or value appear to accelerate and decelerate, maintaining some sense of momentum,
+as moving objects tend to do in the real world. Interrupting these transitions can break this
+illusion, however, as the object's position at the time of interruption then becomes the new
+`from` value for a new transition. The object often abrubtly changes its direction and speed as
+it suddenly jumps to the beginning of the new transition's easing curve. Physics-based animations,
+like the spring-based transition system proposed here, can get around this problem by maintaining
+an animating value's "momentum".
+
```

Comment:
Keeping a bit of momentum is nice not only for programmatic transitions, but also for user interaction, e.g. it could give an extra subtlety to e.g. our orbit controls, when swiping the object spins and stops spinning through dampening.

This would probably be out-of-scope but added in an "ideas for future extension" section towards the end

Reply:
Yeah! I think user-interaction is where physics-based transitions really shine! Making the orbit controls a bit more physics-based would definitely be cool but, I agree, would be a future piece of work. To be clear, anyone who mutates their uniform/attribute values based on user interaction would get that "bit of momentum" we're talking about by using the physics-based spring transition proposed in this RFC.